src/cli/credrank.js

```diff
@@ -73,4 +97,38 @@ function printCredSummaryTable(credGraph: CredGraph) {
   sortedParticipants.slice(0, 20).forEach((n) => console.log(row(n)));
 }
 
+function printCredDiffTable(credGraph: CredGraph, priorCredGraph: CredGraph) {
+  console.log(`# Top Participants By New Cred`);
+  const priorParticipants: Map<Uuid, Participant> = new Map();
+  for (const participant of priorCredGraph.participants())
+    priorParticipants.set(participant.id, participant);
+
+  const credParticipants = Array.from(credGraph.participants());
+  const sortedParticipants = sortBy(credParticipants, (p) => -p.cred);
+  function row({cred, description, id}) {
+    const prior = priorParticipants.get(id);
+    if (!prior)
+      throw `Participant [${description}, ${id}] exists in the new scores but not in the old. Rerun without -d to refresh.`;
```

Comment:
I note that if there's a new participant, it will error, but if a participant disappeared, it will silently elide their removal. Seems inconsistent and like it may result in confusion.

Reply:
added a check for participant count equality